hal/armv7m/imxrt/10xx/106x/imxrt.c

```diff
@@ -683,10 +683,14 @@ void _imxrt_ccmInitEnetPll(u8 enclk0, u8 enclk1, u8 enclk2, u8 div0, u8 div1)
 		enet_pll |= 1 << 21;
 	}
 
-	*(imxrt_common.ccm_analog + ccm_analog_pll_enet) = enet_pll;
+	*(imxrt_common.ccm_analog + ccm_analog_pll_enet) = 1 << 16; /* enable bypass during clock frequency change */
```

Comment:
personally I would prefer to have the comment in the line before (as the comment is for the whole code block until line 693 - and not to create too-long lines unnecessarily), but I'm not enforcing it.

Reply:
good idea for clarity, will apply it